hive-metastore/src/test/java/org/apache/iceberg/hive/HiveMetastoreTest.java

```diff
@@ -38,7 +38,7 @@
   @BeforeClass
   public static void startMetastore() throws Exception {
     HiveMetastoreTest.metastore = new TestHiveMetastore();
-    metastore.start();
+    metastore.start(new HiveConf(HiveMetastoreTest.class));
```

Comment:
Is this change needed? `start()` is still defined and uses `HiveMetastoreTest.class`. The only difference is that this doesn't pass a `Configuration` and the parameterless `start` passes `new Configuration()`.

Reply:
Yeah... that is something I forgot to revert.
Thanks for noticing!

Removed the change.